packages/documentation/pages/components/form.vue

```diff
@@ -271,6 +281,7 @@ export default defineComponent({
 			personalDetails: {
 				firstName: 'John',
 				lastName: 'Smith',
+				fullName: `Smith${Dashes.EmDash}John`,
```

Comment:
I think a hypen is more appropriate here.

Reply:
ok...